features/clear_form.feature

```diff
@@ -163,95 +162,91 @@ Scenario: user can reset a form for Edit App
+	And I should have filled in "a, b, c" for "Student Names"
 
 # Story ID: 153070009
 @javascript
 Scenario: user can reset a form for Edit Iteration
-	Given I am on the the Apps page
+	Given I am on the Apps page
 	And I follow "ESaaS Engagement Tracker"
 	And I follow "2017-09-15"
 	And I follow "Edit"
-	When I fill in "2016-08-15" for "End date:"
-	And I select "1 hour 15 min" from "Duration:"
-	And I select "Mostly disagree" from "Demeanor:"
-	And I select "Mostly agree" from "Engaged:"
-	And I fill in "some comment" for "Engagment comments:"
-	And I select "Neither agree nor disagree" from "Communication:"
-	And I fill in "some comment" for "Communication comments:"
-	And I select "Strongly agree" from "Understanding:"
-	And I fill in "some comment" for "Understanding comments:"
-	And I select "Strongly disagree" from "Effectiveness:"
-	And I fill in "some comment" for "Effectiveness comments:"
-	And I select "Mostly disagree" from "Satisfied:"
-	And I fill in "some comment" for "Satisfied comments:"
-	And I press "reset"
-	Then I should have filled in "2017-09-15" for "End date:"
-	And I should have selected "15 min" from "Duration:"
-	And I should have selected "Strongly agree" from "Demeanor:"
-	And I should have selected "Strongly agree" from "Engaged:"
-	And I should have selected "Strongly agree" from "Communication:"
-	And I should have selected "Strongly agree" from "Understanding:"
-	And I should have selected "Strongly agree" from "Demeanor:"
-	And I should have selected "Strongly agree" from "Effectiveness:"
-	And I should have selected "Strongly agree" from "Satisfied:"
-	And All text fields are empty for the form with selector ".edit_iteration"
\ No newline at end of file
+	When I fill in "2016-08-15" for "End Date"
+	And I select "1 hour 15 min" from "Duration"
+	And I select "Mostly disagree" from "Demeanor"
+	And I fill in "some comment" for "Demeanor Comments"
+	And I select "Mostly agree" from "Engagement"
+	And I fill in "some comment" for "Engagement Comments"
+	And I select "Neither agree nor disagree" from "Communication"
+	And I fill in "some comment" for "Communication Comments"
+	And I select "Strongly agree" from "Understanding"
+	And I fill in "some comment" for "Understanding Comments"
+	And I select "Strongly disagree" from "Effectiveness"
+	And I fill in "some comment" for "Effectiveness Comments"
+	And I select "Mostly disagree" from "Satisfaction"
+	And I fill in "some comment" for "Satisfaction Comments"
+	And I press "Reset"
+	Then I should have filled in "2017-11-27" for "End Date"
+	And the "Duration" field should contain "15 min"
+	And the "Demeanor" field should contain "Strongly agree"
+	And the "Engagement" field should contain "Strongly agree"
+	And the "Communication" field should contain "Strongly agree"
+	And the "Understanding" field should contain "Strongly agree"
+	And the "Effectiveness" field should contain "Strongly agree"
+	And the "Satisfaction" field should contain "Strongly agree"
```

Comment:
These could probably be refactored into the standard hash table format from other feature files, but they are very extensive tests! Good job.

Reply:
agree with Joe here!